inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -0,0 +1,358 @@
+#===============================================================================
+#  input_template.yaml
+#
+#  User inputs for bat-can model creation and running.
+#===============================================================================
+
+cell-description:
+  anode:
+    class: 'dense_electrode'
+    bulk-phase: 'lithium_metal' # Cantera phase name
+    surf-phase: 'lithium_electrolyte' # Cantera phase name
+    electrolyte-phase: 'electrolyte' # Cantera phase name
+    conductor-phase: 'electron' # Cantera phase name
+    stored-ion: # Details on the stored species. Used for capacity calc.
+      name: Li[metal]
+      charge: 1
+    mobile-ion:  'Li+[elyt]' # Species name for Li+ in elyte.
+    thickness: 15e-6 # anode thickness, m
+    minimum-thickness: 1e-9 # minimum anode thickness. 
+    phi_0: 0. # Initial electric potential, V
+    A_surf_ratio: 1.0 # Interface area per unit geometric area
+    C_dl: 6e3 #F/m2
+    dy_elyte: 2e-6
+  separator:
+    class: 'porous_separator'
+    thickness: 20e-6  # separator thickness, m
+    n_points: 5       # Number of finite volumes to discretize
+    electrolyte-phase: 'electrolyte' # Cantera phase name
+    sigma_io: 1.3 # S/m DOI:10.1149/2.0571912jes for 1M LiPF6 in EC:EMC (3:7 w:w) at 50 deg C
+    phi_0: 2.96 # Initial electric potential, V
+    eps_electrolyte: 0.65 # Electrolyte volume fraction, -
+  cathode:
+    class: 'nucleation_single_particle'
+    bulk-phase: 'cathode' # Cantera phase name
+    surf-phase: 'cathode_electrolyte' # Cantera phase name
+    electrolyte-phase: 'electrolyte' # Cantera phase name
+    conductor-phase: 'electron' # Cantera phase name
+    stored-ion: # Details on the stored species. Used for capacity calc.
+      name: Li2O2[cathode]
+      charge: 2
+    mobile-ion:  'Li+[elyt]' # Species name for Li+ in elyte.
+    thickness: 50e-6 # anode thickness, m
+    r_p: 6e-6 # Particle radius, m
+    phi_0: 2.95 # Initial electric potential, V
+    eps_solid: 0.65 # Solid phase volume fraction, -
+    X_0: 'C[cathode]:0.80, Li2O2[cathode]:0.00, Cat[cathode]:0.20'
```

Comment:
I'm not sure what this means, and probably relates back to some of my other comments on the various phase names.  Probably easier to have a convo, off line about it.